testing/test_schools_ks.py

```diff
@@ -0,0 +1,121 @@
+import csv
+import os
+import subprocess
+import sys
+import xml.etree.ElementTree as ET
+from pathlib import Path
+from pprint import pprint
+from sys import stderr
+
+from utility import position_on_edge
+
+if 'SUMO_HOME' in os.environ:
+    tools = os.path.join(os.environ['SUMO_HOME'], 'tools')
+    sys.path.append(tools)
+else:
+    sys.exit("Please declare environment variable 'SUMO_HOME' to use sumolib")
+
+import sumolib
+
+
+class TestInstance:
+    def __init__(self, name: str, net_file: str, gen_stats_in_file: str, real_stats_file: str, centre):
+        self.name = name
+        self.centre = centre
+        self.net_file = net_file
+        self.gen_stats_in_file = gen_stats_in_file
+        self.real_stats_file = real_stats_file
+
+        try:
+            Path(self.net_file).resolve(strict=True)
+            Path(self.gen_stats_in_file).resolve(strict=True)
+            Path(self.real_stats_file).resolve(strict=True)
+        except FileNotFoundError:
+            print(f"Files for test instance: {self.name} does not exist", file=stderr)
+            pprint(self.__dict__)
+            exit(1)
+
+
+test_instances = [
+    TestInstance("aalborg", "../in/cities/aalborg.net.xml", "../in/cities/aalborg.stat.xml",
+                 "../stats/aalborg.stat.xml", "9396,12766"),
+    TestInstance("esbjerg", "../in/cities/esbjerg.net.xml", "../in/cities/esbjerg.stat.xml",
+                 "../stats/esbjerg.stat.xml", "7476,1712"),
+    TestInstance("randers", "../in/cities/randers.net.xml", "../in/cities/randers.stat.xml",
+                 "../stats/randers.stat.xml", "19516,6606"),
+    TestInstance("slagelse", "../in/cities/slagelse.net.xml", "../in/cities/slagelse.stat.xml",
+                 "../stats/slagelse.stat.xml", "6073,4445"),
+    TestInstance("vejen", "../in/cities/vejen.net.xml", "../in/cities/vejen.stat.xml", "../stats/vejen.stat.xml",
+                 "37800,3790")
+]
+
+
+def write_school_coords(net: sumolib.net.Net, stats: ET.ElementTree, filename):
+    """
+    Writes all schools' positions found in stats file to a csv called 'filename'. These coordinates can be used for
+    testing, e.g 2d KS tests between generated schools, and real school positions in the city
+    :param net: network file that the schools in stats file is placed on :param stats: stats
+    file parsed with ElementTree containing schools :param filename: name of csv to be written
```

Comment:
For Pycharm to render this documentation, each param needs to be on it's own line.
```suggestion
    Writes all schools' positions found in stats file to a csv called 'filename'. These coordinates can be used for
    testing, e.g 2d KS tests between generated schools, and real school positions in the city
    :param net: network file that the schools in stats file is placed on 
    :param stats: stats file parsed with ElementTree containing schools 
    :param filename: name of csv to be written
```

Reply:
e6f9816319932fe4522f0d8e224fe8bf2aae6bdf